test/tests/TestRemoveSymlinks.hx

```diff
@@ -0,0 +1,41 @@
+package tests;
+
+import haxe.unit.TestCase;
+
+class TestRemoveSymlinks extends TestCase
+{
+	//----------- properties, fields ------------//
+
+	//--------------- constructor ---------------//
+	public function new()
+	{
+		super();
+	}
+
+
+	//--------------- initialize ----------------//
+
+	override public function setup():Void
+	{
+		Sys.command("neko", ["./bin/haxelib.n", "setup", Sys.getCwd() + "haxelib-repo"]);
+		Sys.command("neko", ["./bin/haxelib.n", "install", "nme-dev", "1.3.2", "--always"]);
```

Comment:
Please do not depends on 3rd party libs in our test. They could take long time to download. 
Create only a minimal lib in `/test/libraries` and use that.